package.json

```diff
@@ -8,12 +8,14 @@
     "ios": "expo start --ios",
     "web": "expo start --web",
     "pretty": "prettier --write .",
-    "python": "./venv/bin/python"
+    "python": "./venv/bin/python",
+    "firestore": "firebase emulators:start --only firestore"
```

Comment:
I connected to Firestore emulator (installed firebase-tools globally and edited firebaseConfig.js to connect), but I got an error (below).

Hess's email about this topic also said something about needing paid plan for Firebase Export (or something like that).

Is it better to remove all the firestore emulator stuff from this PR for now, or should we wait to see if Hess can upgrade the Firebase permissions?

![image](https://github.com/Golf-Drill-Challenge-App/Golf-App/assets/82061589/899b33d4-0d1b-4a86-a257-a024f6507232)

Reply:
yes let's remove the emulator stuff